tilmeld.html

```diff
@@ -28,7 +28,56 @@
     </header>
 
     <main>
+        <section class="banner">
+            <div class="banner__content">
+                <img class="banner__photo" src="assets/images/mobile/tilmeld-banner-mobile.webp"
+                    alt="mere end dyrevenner banner">
+                <div class="overlay__text">
+                    <p class="tilmeld__h1">TILMELD</p>
+                </div>
+            </div>
+        </section>
+        <section class="section-white">
+            <div class="section-white__content">
+                <h3 class="section-white__h3">Sådan foregår kampagnen</h3>
+                <p>Dyrenes Beskyttelse inviterer dig til en hyggedag på dit lokale internat! Her kan du møde både dyrene
+                    og andre unge, der måske har det ligesom dig.
+                    <br>
+                    <br>
+                    Dagen handler om at skabe nye forbindelser – både med pelsede venner og nye bekendtskaber.
+                    <br>
+                    <br>
+                    Ensomhed kan ramme os alle, og det kan være lettere at åbne op med søde kæledyr som icebreakere.
+                    <br>
+                    <br>
+                    Det er en uformel og tryg ramme, hvor alle er velkomne!
+                </p>
+
+            </div>
```

Comment:
Ift. navngivningen af classes tror jeg vi skal fokusere mere på hvad indholdet er i sektionen. Fx. vil jeg kalde den "introduction" fremfor "section-white" fordi det tydligere beskriver hvad der er i sektionen. Ved ikke om det giver mening?
Vær også opmærksom på at du ikke skal indsætte typografi css. Kan se at du har lavet en class til h3 tekst. Kig i vores base-style.css for at se hvilke typografier du kan vælge imellem. Der har jeg navngivet og lavet alle dem som kommer til at være på vores sider. Vi elsker at genbruge dem, så vi ikke skal skrive koden dobbelt til flere forskellige sider. 
Ift. p delen er det næsten fyfy i kode verdenen at bruge <br>. I Figma kan du se at hver tekst er i deres egen p tag så vi selv kan styre mellemrummet i det div element som omslutter dem. 


```suggestion
        <section class="introduction">
            <div class="introduction__content">
                <h3>Sådan foregår kampagnen</h3>
                <div>
                    <p>Dyrenes Beskyttelse inviterer dig til en hyggedag på dit lokale internat! Her kan du møde både dyrene
                    og andre unge, der måske har det ligesom dig.  </p>
                    
                    <p> Dagen handler om at skabe nye forbindelser – både med pelsede venner og nye bekendtskaber. </p>
                    
                    <p> Ensomhed kan ramme os alle, og det kan være lettere at åbne op med søde kæledyr som icebreakere.</p>
                    
                    <p> Det er en uformel og tryg ramme, hvor alle er velkomne! </p>
                    
                </div>
            </div>
```